src/app/saga.ts

```diff
@@ -106,6 +113,9 @@ const TAG = 'app/saga'
 //    (which will put an app into `background` state until dialog disappears).
 const DO_NOT_LOCK_PERIOD = 30000 // 30 sec
 
+// REVIEW_INTERVAL is the time between app review prompts
+const REVIEW_INTERVAL = ONE_DAY_IN_MILLIS * 91 // 91 days
```

Comment:
what's the benefit of having a 90 day window here? android users potentially get this prompt 4 times per year, but iOS users will get this on month 0, 3, 6, and then nothing for 6 months? why not have 120 days or so, so that the prompt is evenly spaced?

Reply:
Changed to 120 days in 6a154e6f519154f38ba1c3dfb3d77540ff961fa4!